runtime/oti/JITInterface.hpp

```diff
@@ -192,8 +187,23 @@ class VM_JITInterface
 			jitVTableIndex = 0 - (instruction & 0x00000FFF);
 		}
 #elif defined(J9VM_ARCH_AARCH64)
-		/* TODO: Implement this */
-		assert(0);
+		/* Virtual call instructions
+		 *	...
+		 *	movn x9, vTableOffset
+		 *	ldr x9, [register, x9]
```

Comment:
Please make it clear that `register` is actually the `vftRegister`.

Reply:
Modified the comment to `vftReg`.